modules/invoice/api/invoice-routes.ts

```diff
@@ -3,10 +3,92 @@ import * as core from 'express-serve-static-core';
 import { getQtyFromRequest } from '../../../utils/route-utils';
 import getRandomInvoices from '../utils/getRandomInvoices';
 
+/**
```

Comment:
Please could you move this definition to file of the object in 'consts' folder please :)

Reply:
Sure. It's done :D